packages/panzer/disc-fe/src/Panzer_L2Projection_impl.hpp

```diff
@@ -258,25 +259,34 @@ namespace panzer {
     }
     PHX::exec_space::fence();
 
-    ghostedMatrix->fillComplete();
-    const auto exporter = factory.getGhostedExport(0);
-    ownedMatrix->doExport(*ghostedMatrix, *exporter, Tpetra::ADD);
-    ownedMatrix->fillComplete();
-
+    {
+      PANZER_FUNC_TIME_MONITOR("Exporting of mass matrix");
+      ghostedMatrix->fillComplete();
+      const auto exporter = factory.getGhostedExport(0);
+      ownedMatrix->doExport(*ghostedMatrix, *exporter, Tpetra::ADD);
+      ownedMatrix->fillComplete();
+    }
     return ownedMatrix;
   }
 
   template<typename LO, typename GO>
   Teuchos::RCP<Tpetra::MultiVector<double,LO,GO,Kokkos::Compat::KokkosDeviceWrapperNode<PHX::Device>>>
```

Comment:
In some places, you use Tpetra's default Node type (by not specifying the Node template parameter).  Here, you specify the Node type explicitly.  Pick one (preferably the former), or risk incompatibility.